packages/common/src/access/can-edit-dataset.ts

```diff
@@ -0,0 +1,13 @@
+import { IItem, IUser } from "@esri/arcgis-rest-types";
+import { hasBasePriv } from "./has-base-priv";
+import { canEditItem } from "./can-edit-item";
+
+/**
+ *
+ * @param {IItem} model
+ * @param {IUser} currentUser
```

Comment:
should this be `username` for both consistency and also the method can be used to check if another user (not current) is able to edit a dataset. For example a table could show all members of a group and an indication if that user can edit this dataset.

Reply:
that's true, it should be `user`